.github/workflows/deploy-docs.yaml

```diff
@@ -1,18 +1,27 @@
-name: Deploy docs to GH pages
+name: Deploy docs
 
 on:
   push:
-    branches: ["main"]
+    branches:
+      - "main"
+    tags:
+      - "*.*.*"
+      - "v*.*.*"
     paths:
       - 'docs/**'
       - '.github/workflows/deploy-docs.yaml'
   workflow_dispatch:
-
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
-permissions:
-  contents: read
-  pages: write
-  id-token: write
+    inputs:
+      version:
+        type: string
+        description: "Version of the documentation to deploy"
+        required: false
+        default: "latest"
```

Comment:
It is just an alias set to the latest deployed version?
Asking just in case it should've been a variable with semver here.

Reply:
Yeah, it would work as follows:
* `/latest` would always point to the version of the book that is in our trunk (`main` branch)
* Versions for tags will be deployed to `/x.y.z` and kept until we manually remove them